optimize/util/optimize-commons/src/main/java/io/camunda/optimize/upgrade/os/OpenSearchClientBuilder.java

```diff
@@ -111,21 +111,25 @@ private static OpenSearchTransport getAwsTransport(final ConfigurationService os
             .build());
   }
 
-  private static boolean isAws() {
-    AwsCredentialsProvider credentialsProvider = DefaultCredentialsProvider.create();
-    try {
-      credentialsProvider.resolveCredentials();
-      log.info("AWS Credentials can be resolved. Use AWS OpenSearch");
-      return true;
-    } catch (Exception e) {
-      log.info("Use standard OpenSearch since AWS not configured ({}) ", e.getMessage());
-      return false;
+  private static boolean useAwsCredentials(final ConfigurationService configurationService) {
+    if (configurationService.getOpenSearchConfiguration().getConnection().getAwsEnabled()) {
+      final AwsCredentialsProvider credentialsProvider = DefaultCredentialsProvider.create();
+      try {
+        credentialsProvider.resolveCredentials();
+        log.info("AWS Credentials can be resolved. Use AWS OpenSearch");
+        return true;
+      } catch (final Exception e) {
+        log.info("Use standard OpenSearch since AWS not configured ({}) ", e.getMessage());
+        return false;
+      }
     }
+    log.info("Not using AWS credentials for OpenSearch");
```

Comment:
what about using `"AWS Credentials are disabled. Using basic auth."` to be consistent with Operate?